tilecloud_chain/views/admin.py

```diff
@@ -51,16 +51,22 @@ def __init__(self, command: List[str]):
     def run(self):
         try:
             display_command = ' '.join([shlex.quote(arg) for arg in self.command])
-            LOG.error("Run the command `{}`".format(display_command))
+            LOG.info("Run the command `{}`".format(display_command))
             with subprocess.Popen(
                 self.command,
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
             ) as process:
                 process.wait()
```

Comment:
Note from the python doc:

> Note: This will deadlock when using stdout=PIPE or stderr=PIPE and the child process generates enough output to a pipe such that it blocks waiting for the OS pipe buffer to accept more data. Use Popen.communicate() when using pipes to avoid that.